Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="234" judge-id="234" status="SE" scoring="ACM" run-tests="1" contest-id="3208" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>13377c73-8269-4d9c-b1e1-32100a02cec8</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="284" real-time="300" max-memory-used="18698240" >
  <program-stats-str>{ utime=219, stime=65, ptime=284, rtime=300, maxvsz=18698240, maxrss=19476480, nvcsw=1, nivcsw=47 }</program-stats-str>
      <input size="10">1 2 3 4 5
</input>
      <output size="2">a=</output>
      <stderr size="327">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000234&quot;, line 5, in &lt;module&gt;
    for a in range(int(input('a='))):
ValueError: invalid literal for int() with base 10: '1 2 3 4 5'
</stderr>
    </test>
  </tests>
</testing-report>
